src/app/shared/sidebar/sidebar-dropdown.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="setting-option mb-3 p-3">
-  <h4><label for="{{id}}">{{label | translate}}</label></h4>
+<h4><label for="{{id}}">{{label | translate}}</label></h4>
```

Comment:
@GauravD2t : Can you **add back in** the space you removed from here? The alignment is now messed up in this file as the `<h4>` should be under the `<div>`. We want to avoid reformatting code unless it's necessary as it makes the PR very messy to review.